src/data/finset/basic.lean

```diff
@@ -2183,7 +2183,7 @@ begin
   exact hf heq ▸ hy,
 end
 
-lemma filter_mem_image_eq_image (f : α → β) (s : finset α) (t : finset β) (h : ∀ x ∈ s, f x ∈ t) :
```

Comment:
Was this a bad find/replace?

Reply:
Yes, exactly so. Ideally `mem_image` would be named `mem_image_iff` but fixing that would be more work, and outside the scope of this.